### DISCUSSION TITLE
Weired suggestions starting with `[]: #`

### DISCUSSION THREAD

**Discussion by PRO-2684** [community/community#119392] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Copilot sometimes suggest some weired content related to current workspace, which seems like internal control sequences. For me, this phenomenon mostly occurs when:

- Working with Markdown files
- Language is Chinese
- There's another tab with corresponding English content
- I'm at the end of the Markdown file

![image](https://github.com/community/community/assets/54608551/5ba7b550-ed39-4055-bd02-ff72fb01e8fe)

**Comment by thanachotnual** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> Copilot sometimes suggest some weired content related to current workspace, which seems like internal control sequences. For me, this phenomenon mostly occurs when:
> 
> - Working with Markdown files
> - Language is Chinese
> - There's another tab with corresponding English content
> - I'm at the end of the Markdown file
> 
> ![image](https://github.com/community/community/assets/54608551/5ba7b550-ed39-4055-bd02-ff72fb01e8fe)
>
